index.js

```diff
@@ -12,10 +12,13 @@ exports.register = function () {
   this.route_list={};
 
   this.load_rcpt_to_routes_ini();
-  this.merge_redis_ini();
+  if (this.cfg.redis.opts.enabled) {
+    this.merge_redis_ini();
+    this.redis_ping();
```

Comment:
Did I add this line back arbitrarily?  Does it need to be removed?

Reply:
Line 17, that is

Reply:
Huh, it appears you did.